src/go/rpk/pkg/tuners/factory/factory.go

```diff
@@ -42,6 +43,7 @@ var (
 		"swappiness":            (*tunersFactory).newSwappinessTuner,
 		"transparent_hugepages": (*tunersFactory).newTHPTuner,
 		"coredump":              (*tunersFactory).newCoredumpTuner,
+		"write_cache":           (*tunersFactory).newGcpWriteCacheTuner,
```

Comment:
The yaml field is rpk.tune_disk_write_cache. Please change this to disk_write_cache, to be more consistent with the others, and move it to the top with the other disk_* tuners.